monitoring/mongodb/alerts.yaml

```diff
@@ -183,7 +183,7 @@ groups:
 
   - alert: MongoDbRSNotSynced
     expr: |
-      sum by (rs_nm) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", member_state="SECONDARY"}) != (${replicas} - 1)
+      count by (rs_nm, instance) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", member_state="SECONDARY"}) != (${replicas} - 1)
```

Comment:
what is `instance` ?

Reply:
the instance on the metrics is tipically the `IP:PORT => instance="10.233.226.186:9216"`

Reply:
if we count by instance, we will only get series of 1 element (each instance)...

Reply:
No , when we count by the instance we indeed have the right number cf ( this was done on a 6 nodes ) : 
![image](https://github.com/user-attachments/assets/043f2b93-f635-4c7e-af2b-9ec55c86db66)

Reply:
No the count by the instance returns indeed the right number of secondary members cf : ( this was done on a 6 nodes artesca ) 
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/22d2063d-2e3e-4100-96a8-ec02367827ff">

Reply:
but why does it work? AFAIK, each pod should be a single instance, something fishy here...

can you share the actual metrics: e.g. in explore, check which metrics series we get with

```
mongodb_rs_members_state{namespace="zenko", pod=~"data-db-mongodb_sharded-shard0-data"}
```

Reply:
with : 
`mongodb_rs_members_state{namespace="zenko", pod=~"data-db-mongodb-sharded-shard0-data.*"}`\

this is what is returned : 
`{__name__="mongodb_rs_members_state", cl_id="65d9fdecfdcca86bb7cd2b98", container="metrics", endpoint="metrics", instance="10.233.178.136:9216", job="zenko/data-db-mongodb-sharded-shard0-data", member_idx="data-db-mongodb-sharded-shard0-data-1.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017", member_state="SECONDARY", namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", prometheus="metalk8s-monitoring/prometheus-operator-prometheus", rs_nm="data-db-mongodb-sharded-shard-0", rs_state="1"}`


and the value returned is 2 ( the state as it's a secondary ) => 
![image](https://github.com/user-attachments/assets/745c5e70-574f-4792-826a-57177861ef50)

Reply:
as we cam see in the "full" metrics, each pod reports the status (primary/secondary/...) of every member of the cluster participating in the replicaset, as identified by the `member_idx` label